Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
I think I like this approach a lot better.  What are your results if you run with this?